src/i18n.ts

```diff
@@ -1,5 +1,12 @@
 import Vue from 'vue';
 import VueI18n from 'vue-i18n';
+import en from '@/locales/index.ts';
```

Comment:
You can remove all import and just use 
```suggestion
import messages from '@/locales';
```

Reply:
Well, actually now it's not doing the translations anymore. So there's still a line missing somewhere?